sysutils/ondir/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             ondir
+version          0.2.3
+revision         1
+
+categories       sysutils
+platforms        darwin
+license          GLP-2
```

Comment:
I am assuming that the `license` is actually GPL-2?

Reply:
that's correct.
it is actually the content of the COPYING file (which is installed in the share/doc director).